webpack/assets/javascripts/react_app/components/common/Timer.js

```diff
@@ -0,0 +1,20 @@
+class Timer {
```

Comment:
Ok, i still think that it might belong more in reacrt_app/common, since it does not really classify as a ui component in my mind.